fabric/dataclient.go

```diff
@@ -335,6 +340,30 @@ func getEndpointsByService(namespace, name, protocol string, servicePort *servic
 	}
 }
 
+// dummy
+func getStacksetTraficByName(namespace, name string) []*zv1.ActualTraffic {
```

Comment:
```suggestion
func getStacksetTrafficByName(namespace, name string) []*zv1.ActualTraffic {
```

Reply:
I will change it on my PR, because I have some not committed changes, that I hope to push in the evening today or tomorrow